src/core/Akka.Persistence.TCK/Query/PersistenceIdsSpec.cs

```diff
@@ -247,7 +247,7 @@ protected IActorRef WriteSnapshot(string persistenceId, int n)
                 ExpectMsg($"{persistenceId}-{i}-done");
             }
 
-            var metadata = new SnapshotMetadata(persistenceId, n + 10);
+            var metadata = new SnapshotMetadata(persistenceId, n + 10, Sys.Scheduler.Now.DateTime);
```

Comment:
Standard call we're going to use to get the current UTC time from the `ITimeProvider` built into the `IScheduler`.